### PR TITLE
Opsgenie intergration

### PR DESCRIPTION
**Pull request by pampatzoglou**
Related to https://github.com/w3f/infrastructure/issues/16 
Closes https://github.com/w3f/infrastructure/issues/16

### CODE REVIEW

**Comment by pampatzoglou** [comment]
This job has been blocked because Docker Layer Caching is not available on your plan. Please upgrade to continue building.

File: README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ be able to install, together with `base-services` and `loki`, these stable chart
   `Infrastructure` vault on 1Password in an item called `Grafana` and the
   Digital Ocean access token and spaces keys in the `DigitalOcean API credentials`
   item. The Cloudflare API key is available from Cloudflare's web UI.
+  The opsgenie token is available in opsgenie's intergration page. The default opsgenie API is https://api.opsgenie.com for international deployments and https://api.eu.opsgenie.com for EU deployments.
```
Comment (fgimenez) [inline README.md]: We need to define here which environment variable we need.

File: scripts/integration-tests.sh [hunk 2]
```diff
@@ -6,6 +6,47 @@ source /scripts/bootstrap-helm.sh
...
+
+}
+
+intergration_test_sequence(){
+  echo Running intergration tests...
+  alertmanager_intergration
+
+}
+
+wait_cluster_ready(){
```
Comment (fgimenez) [inline scripts/integration-tests.sh]: We are not using this function.

File: scripts/integration-tests.sh [hunk 3]
```diff
@@ -6,6 +6,47 @@ source /scripts/bootstrap-helm.sh
 
 run_tests() {
     echo Running tests...
+    intergration_test_sequence
```
Comment (fgimenez) [inline scripts/integration-tests.sh]: Typo
```suggestion
    integration_test_sequence
```

File: scripts/integration-tests.sh [hunk 4]
```diff
@@ -6,6 +6,47 @@ source /scripts/bootstrap-helm.sh
...
+
+}
+
+intergration_test_sequence(){
```
Comment (fgimenez) [inline scripts/integration-tests.sh]:
```suggestion
integration_test_sequence(){
```

File: scripts/integration-tests.sh [hunk 5]
```diff
@@ -6,6 +6,47 @@ source /scripts/bootstrap-helm.sh
...
+  echo Running intergration tests...
```
Comment (fgimenez) [inline scripts/integration-tests.sh]:
```suggestion
  echo Running integration tests...
```

File: scripts/integration-tests.sh [hunk 6]
```diff
@@ -6,6 +6,47 @@ source /scripts/bootstrap-helm.sh
...
+  alertmanager_intergration
```
Comment (fgimenez) [inline scripts/integration-tests.sh]:
```suggestion
  alertmanager_integration
```

**Comment by fgimenez** [comment]
Please add `Closes https://github.com/w3f/infrastructure/issues/16` to the description

File: README.md (hunk 1 above)
Comment (fgimenez) [reply README.md]: We still need `OPSGENIE_TOKEN` in the readme, where it talks about the env vars. Is it already defined in circleci?

Comment (pampatzoglou) [reply README.md]: > We still need `OPSGENIE_TOKEN` in the readme, where it talks about the env vars. Is it already defined in circleci?

Yes its defined in circleCI in w3f>engineering-base-services

**Pull request merged**